ui/components/AutoComplete.js

```diff
@@ -0,0 +1,41 @@
+/* eslint-disable react/prop-types */
+import * as React from "react";
+import Downshift from "downshift";
+
+export const AutoComplete = ({ items, handleSelectionChange }) => {
+  return (
+    <div>
+      <Downshift onChange={handleSelectionChange} itemToString={(item) => (item ? item.value : "")}>

```

Comment:
its better to have a validation for `item` as to whether it is a string or not.

Reply:
Uhm, can you be more explicit?

Reply:
Never mind. This is not that important for now.